reference/api/search.md

```diff
@@ -229,7 +231,7 @@ Query terms enclosed in double quotes are treated as [phrase searches](#query-q)
 
 ## Search parameters
 
-Here follows an exhaustive description of each search parameter currently available when using the search endpoint. Unless otherwise noted, all parameters are valid for both `GET` and `POST` routes.
+Here follows an exhaustive description of each search parameter currently available when using the search endpoint. Unless otherwise noted, all parameters are valid for the `GET`, `POST`, and `/multi-index` routes.
```

Comment:
```suggestion
Here follows an exhaustive description of each search parameter currently available when using the search endpoint. Unless otherwise noted, all parameters are valid for the `GET`, `POST` for `/indexes/:indexUid/search`, and POST `/multi-search` routes.
```